cluster-autoscaler/processors/datadog/common/common.go

```diff
@@ -61,7 +68,22 @@ func SetNodeLocalDataResource(nodeInfo *schedulerframework.NodeInfo) {
 	if node.Status.Capacity == nil {
 		node.Status.Capacity = apiv1.ResourceList{}
 	}
-	node.Status.Capacity[DatadogLocalDataResource] = DatadogLocalDataQuantity.DeepCopy()
-	node.Status.Allocatable[DatadogLocalDataResource] = DatadogLocalDataQuantity.DeepCopy()
+
+	provisioner, _ := node.Labels[DatadogLocalStorageProvisionerLabel]
+	switch provisioner {
+	case "topolvm":
```

Comment:
I think we can either add "openebs-lvm" as well here or pass down the provisioning mode field to the node themselves, but we can do that in another PR

Reply:
I'll do 👍

Reply:
https://github.com/DataDog/autoscaler/pull/53